src/game/Laptop/IMP_Begin_Screen.cc

```diff
@@ -289,6 +300,20 @@ static void BtnIMPBeginScreenDoneCallback(GUI_BUTTON *btn, INT32 reason)
 			iCurrentImpPage = IMP_MAIN_PAGE;
 			fButtonPendingFlag = TRUE;
 		}
+
+		if(GCM->getGamePolicy()->imp_load_saved_merc_by_nickname)
+		{
+			if(IMPSavedProfileDoesFileExist(pNickNameString.c_str()))
```

Comment:
This is quite a bit different from  `90210` or JA2 1.13, where the nickname is to be typed in place of the XEP624 passcode. 

It would be nicer if we can skip the entire creation process, without even showing the first screen.

Reply:
Good idea, but we need to also increase the char limit for it to work. No biggie.

Reply:
I would like the flow to be (optionally) automatic so that a player could jump right back in after failing the solo/duo straight-for-Meduna scenario.